src/types/ErrorsMessages.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorsMessages {
+  Load = 'Unable to load a todos',
+  Post = "can't send post request",
+  Update = 'Unable to update a todo',
+  Delete = 'Unable to delete a todo',
+  Title = "Title can't be empty",
```

Comment:
Consider using more descriptive error name. For example, instead of `Title`, use `TitleIsRequired`. 📝